src/test/java/screens/GraphSceneTest.java

```diff
@@ -17,11 +18,14 @@
 import org.mockito.runners.MockitoJUnitRunner;
 
 import javax.swing.*;
+import java.lang.reflect.Method;
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
```

Comment:
Thanks for pointing that out! IntelliJ was replacing multiple imports with star imports, but this fixes it.